.github/ISSUE_TEMPLATE/feature_request.yml

```
@@ -0,0 +1,110 @@
+    id: implementation
+    attributes:
+      label: Implementation Ideas
+      description: If you have ideas about how this could be implemented, please share
+      placeholder: Any technical details or implementation suggestions...
+
+  - type: textarea
+    id: additional
+    attributes:
+      label: Additional Context
+      description: Add any other context about the feature request
+      placeholder: Screenshots, mockups, examples, etc.
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)